contracts/contracts/eosio.system/src/voting.cpp

```diff
@@ -119,11 +119,14 @@ namespace eosiosystem {
    double stake2vote( int64_t staked, time_point vote_mature_time ) {
       check(vote_mature_time != time_point(), "vote should have mature time");
 
-      const auto useconds_to_mature = fmax((vote_mature_time - current_time_point()).count(), 0.0);
-      const auto rem_weight = 1.0 - useconds_to_mature / eosio::days( system_contract::vote_mature_period ).count();
+      const auto seconds_to_mature = std::max( (vote_mature_time - current_time_point()).to_seconds(), 0LL );
+      const auto rem_weight = 1.0 - seconds_to_mature / eosio::days( system_contract::vote_mature_period ).to_seconds();
```

Comment:
seconds_to_mature / eosio::days( system_contract::vote_mature_period ).to_seconds() is integer so rem_weight always will be either 0 or 1